validate_depctrl.py

```diff
@@ -59,15 +63,22 @@ def check_changelog(lines, changelog, filename):
 
 def check_contents(entry, basename):
     all_fine = True
-    for ch in entry['channels'].values():
-        for f in ch['files']:
-            filename = basename + f['name']
-            with open(filename, 'rb') as fo:
+    for ch in entry["channels"].values():
+        for f in ch["files"]:
+            filename = basename + f["name"]
+            with open(filename, "rb") as fo:
                 content = fo.read()
-                content_lines = content.decode("utf-8").replace("\r", "").split("\n")
-            hash_good = check_hash(content, f['sha1'], filename)
-            version_good = check_version(content_lines, ch['version'], filename)
-            changelog_good = check_changelog(content_lines, entry['changelog'], filename)
+                try:
+                    content_lines = (
+                        content.decode("utf-8").replace("\r", "").split("\n")
+                    )
```

Comment:
i use a max line length of 100, so this should not be multilined

Reply:
if you are using vscode, maybe it would be better if you add .vscode/settings.json to inform all contributers about your the project formatting settings. for example I set this for my workspace to be identical to your environmental preferences:
![image](https://user-images.githubusercontent.com/25867589/188735218-afbf9aac-1b16-476d-880d-17ca55ecd3db.png)